operator/pkg/helmreconciler/reconciler.go

```diff
@@ -276,7 +276,7 @@ func (h *HelmReconciler) Delete() error {
 	}
 	// Delete IOP with revision:
 	// for this case we update the status field to pending if there are still proxies pointing to this revision
-	// and we do not prune shared resources, same effect as `istioctl uninstall --revision foo` command.
```

Comment:
why delete this?

Reply:
`istioctl uninstall` command still exists @xichengliudui

Reply:
sorry, I thought the command was removed

Reply:
why does this command not exist in version 1.11+?